Misc/holo_world/README.md

```diff
@@ -30,7 +30,7 @@ It consists of fhir data for the EHR FHIR server, Storage FHIR server and some d
 - Configure `config.cfg`
     - Additional details on configuration can be found in the tool's README.md
 - Insert the demo data
-    - `python az_blob_tool.py - upload-folder holograms ./../holorepo_demo_data/storage_glb`
+    - `python az_blob_tool.py - upload-folder holograms ../../holo_world/storage_glb`
```

Comment:
First line may as well be something like `# Holo World: How to set up a complete HoloRepository demo infrastructure` or  anything that includes the "Holo World" title

Reply:
Changed

Reply:
I know, I alrady merged it 2 hours ago 💃